test/libsolidity/semanticTests/functionCall/eof/calling_nonexisting_contract_throws.sol

```diff
@@ -0,0 +1,25 @@
+// TODO: This test should fail on calling a function which pointer was never initialized but for EOF it's impossible to check that
+// there is non-empty code under an address. This should be fixed in with `ISCONTRACT` opcode or any other solution.
```

Comment:
I'd remove this and just call the new test `calling_nonexisting_contract.sol`. Not throwing is the expected behavior for now and I'm not sure we can assume we'll get `ISCONTRACT` eventually (wasn't that changed to `EXTCODETYPE`?).

I'd just add a comment over `d.g()` saying that on legacy that would throw to make the purpose of the test clear.

Reply:
done